functions/Install-DbaSqlWatch.ps1

```diff
@@ -0,0 +1,231 @@
+#ValidationTags#CodeStyle,Messaging,FlowControl,Pipeline#
+function Install-DbaSqlWatch {
+    <#
+    .SYNOPSIS
+        Installs or updates SqlWatch.
+
+    .DESCRIPTION
+        Downloads, extracts and installs or updates SqlWatch.
+        https://sqlwatch.io/
+
+    .PARAMETER SqlInstance
+        SQL Server name or SMO object representing the SQL Server to connect to.
+
+    .PARAMETER SqlCredential
+        Login to the target instance using alternative credentials. Windows and SQL Authentication supported. Accepts credential objects (Get-Credential)
+
+    .PARAMETER Database
+        Specifies the database to install SqlWatch into.s
+
+    .PARAMETER LocalFile
+        Specifies the path to a local file to install SqlWatch from. This *should* be the zipfile as distributed by the maintainers.
+        If this parameter is not specified, the latest version will be downloaded and installed from https://github.com/marcingminski/sqlwatch
+
+    .PARAMETER Force
+        If this switch is enabled, SqlWatch will be downloaded from the internet even if previously cached.
+
+    .PARAMETER Confirm
+        Prompts to confirm actions
+
+    .PARAMETER WhatIf
+        Shows what would happen if the command were to run. No actions are actually performed.
+
+    .PARAMETER EnableException
+        By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+        This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+        Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+    .NOTES
+        Tags: SqlWatch
+        Author: Ken K (github.com/koglerk)
+        Website: https://sqlwatch.io
+        Website: https://dbatools.io
+        Copyright: (c) 2018 by dbatools, licensed under MIT
+        License: MIT https://opensource.org/licenses/MIT
+
+    .LINK
+        https://dbatools.io/Install-DbaSqlWatch
+
+    .EXAMPLE
+        Install-DbaSqlWatch -SqlInstance server1 -Database master
```

Comment:
Since it defaults to master, `-Database master` isnt required. Btw, I removed the Mandatory since it defaulted to master.

Reply:
Updated.